.windsurf/rules/project-context-structure-and-formatting.md

```
@@ -52,6 +52,49 @@
+└── changelog/
+   ├── chainstack-updates-may-30-2025.mdx
+   ├── chainstack-updates-april-1-2025.mdx
+   └── [previous entries...]
+```
+
+### Process
+
+1. **Update `changelog.mdx`**: Copy the previous entry within `<Update...> </Update>` tags and paste it on top of the previous entry. Edit dates and content appropriately.
+
+Example:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)